src/components/modal/privateModal/privateModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback } from 'react'
+
+import { LockOutlined } from '@ant-design/icons'
+import { Form } from 'antd'
+
+import Input from '../../uielements/input'
+import { StyledModal } from './privateModal.style'
+
+interface Props {
```

Comment:
Pls change `interface` to `type`. You can't know it, but we have decided to use `type` (not `interface`) anymore. See https://gitlab.com/thorchain/bepswap/bepswap-web-ui/-/issues/325